plugins/VstEffect/VstEffectControls.cpp

```diff
@@ -558,11 +558,6 @@ ManageVSTEffectView::~ManageVSTEffectView()
 		vstKnobs = nullptr;
 	}
 
-	if( m_vi2->knobFModel != nullptr )
-	{
-		delete [] m_vi2->knobFModel;
-		m_vi2->knobFModel = nullptr;
-	}
```

Comment:
This corresponds to `m_vi2->knobFModel.clear()`, which may still be necessary. (Cleaning up a different class in the destructor like this is disgusting, but it's what we've got for now. I have a rewrite in the works.)